shared/oval/dir_perms_etc_httpd_conf.xml

```diff
@@ -3,10 +3,10 @@
     <metadata>
       <title>Directory /etc/httpd/conf/ Permissions</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 6</platform>
+        <platform>multi_platform_rhel</platform>
       </affected>
       <description>Directory permissions for /etc/httpd/conf/ should be set to 0750 (or stronger).</description>
-      <reference source="JL" ref_id="20140403" ref_url="test_attestation"/>
+      <reference source="galford" ref_id="20160120" ref_url="test_attestation"/>
     </metadata>
     <criteria>
       <criterion test_ref="test_dir_perms_etc_httpd_conf" />
```

Comment:
The full form of the criteria is as below:
```
    <criteria>
      <criterion test_ref="test_dir_perms_etc_httpd_conf" />
    </criteria>
```
There should be added check if the ```httpd``` package is actually installed (because it returns ```fail``` even in the case ```httpd``` isn't installed on the system). This is NOT to say there's bug in this OVAL, but that it should be enhanced for both RHEL-6 && RHEL-7.